server/client.go

```diff
@@ -3493,8 +3493,15 @@ func isReservedReply(reply []byte) bool {
 	if isServiceReply(reply) {
 		return true
 	}
+	rLen := len(reply)
 	// Faster to check with string([:]) than byte-by-byte
-	if len(reply) > gwReplyPrefixLen && string(reply[:gwReplyPrefixLen]) == gwReplyPrefix {
+	if rLen > jsAckPreLen+1 && string(reply[:jsAckPreLen]) == jsAckPre {
```

Comment:
We should not allow clients to use that prefix at all.

Reply:
And need to make sure its API prefix aware, etc. Not sure if that is applicable.

Reply:
not applicable